drivers/bms_ic/bq769x2/bq769x2_registers.h

```diff
@@ -500,6 +500,28 @@ union bq769x2_reg_mfg_status {
     uint16_t u16;
 };
 
+union bq769x2_reg_bat_status {
+    struct
+    {
+        uint16_t CFGUPDATE : 1;
+        uint16_t PCHG_MODE : 1;
+        uint16_t SLEEP_EN : 1;
+        uint16_t POR : 1;
+        uint16_t WD : 1;
+        uint16_t COW_CHK : 1;
+        uint16_t OTPW : 1;
+        uint16_t OTPB : 1;
+        uint16_t SEC : 2;
+        uint16_t FUSE : 1;
+        uint16_t SS : 1;
+        uint16_t PF : 1;
+        uint16_t SD_CMD : 1;
+        uint16_t RSVD_0 : 1;
```

Comment:
The descriptions in `Table 6-2. 0x12 Battery Status() Bit Definitions` are as follows:
![image](https://github.com/LibreSolar/bms-firmware/assets/32713472/ee7bcc5d-1220-4819-9556-71b4e6435a80)
Is the naming of these items intended to differ from what is specified in the data sheet?

Reply:
Looks like the reference manual is inconsistent. I considered below table:
![image](https://github.com/LibreSolar/bms-firmware/assets/17674105/bd73ccf4-3075-4c5e-b91a-362534ea0837)